.buildkite/dss_scaling/pipeline.yml

```diff
@@ -1,5 +1,5 @@
 env:
-  JULIA_VERSION: "1.10.0"
+  JULIA_VERSION: "1.10.5"
```

Comment:
Is this env variable needed?

Reply:
Good question, I'm not sure. Regardless, I think I'd rather not leave it outdated 🤷🏻‍♂️

Reply:
Actually, this pipeline is pretty old, I'm going to update a few things and use climacommon.